### PR TITLE
fix(nextjs): Fix order, and wrong number call to createWebpackConfig()

### PR DESCRIPTION
**Pull request by tvdoan**
## Current Behavior
Assets are not copied to the dist folder after running @nx/next:build

## Expected Behavior
Assets should be copied to the dist folder after @nx/next:build

## Related Issue(s)
https://github.com/nrwl/nx/issues/26900

Fixes #
Fix the order, and wrong number of parameters call to createWebpackConfig()

### CODE REVIEW

File: packages/next/plugins/with-nx.ts [hunk 1]
```diff
@@ -221,8 +226,8 @@ function withNx(
             ? joinPathFragments(workspaceRoot, projectDirectory)
             : workspaceRoot,
           projectDirectory,
-          _nextConfig.nx?.fileReplacements || options.fileReplacements,
-          _nextConfig.nx?.assets || options.assets
+          _nextConfig.nx?.assets || options.assets,
+          _nextConfig.nx?.fileReplacements || options.fileReplacements
```
Comment (tvdoan) [inline packages/next/plugins/with-nx.ts:230]: @ndcunningham The order is incorrect here.  `createWebpackConfig()` expects assets to be the last argument.  Please see here https://github.com/nrwl/nx/blob/6d83ae2fb1806dbc3f6505cdd7d682ae4b662169/packages/next/src/utils/config.ts#L19

**Review by ndcunningham — APPROVED** [review]
LGTM

**Comment by alexandr2110pro** [comment]
Hey guys. When can we expect it to be released?

**Comment by jaysoo** [comment]
19.6 should be today

**Pull request merged**